bugbug/model.py

```diff
@@ -147,7 +147,7 @@ def __init__(self, lemmatization=False):
         else:
             self.text_vectorizer = TfidfVectorizer
 
-        self.cross_validation_enabled = True
```

Comment:
Could you please enable back the cross-validation?

Reply:
Interestingly, there is no hang this time.